cloud_governance/common/pandas/pandas_operations.py

```diff
@@ -0,0 +1,47 @@
+import tempfile
+
+import pandas as pd
+
+from cloud_governance.common.clouds.aws.s3.s3_operations import S3Operations
+
+
+class PandasOperations:
+    """
+    This class performs the pandas operations
+    """
+    CHUNK_SIZE = 5000
+
+    def __init__(self, region_name: str = 'us-east-1'):
+        self.__s3_operations = S3Operations(region_name=region_name)
+
+    def get_dataframe_from_csv_file(self, file_path: str):
```

Comment:
Please add typecheck to all the methods in PandasOperations class

Reply:
@ebattat Adde the AbstarctAthenaOperations for the PyAthenaOperations, Boto3ClientAthenaOperations